### ISSUE TITLE
wp acf export --all does not display path to select

### ISSUE THREAD

**Issue by elvismdev** [hoppinger/advanced-custom-fields-wpcli#72]
When attempting to export all the field-groups with `wp acf export --all` command, it doesn't display the numbered options list with theme paths to select where to export.

![screen shot 2017-03-07 at 12 29 09 pm](https://cloud.githubusercontent.com/assets/3847077/23678093/869d77aa-0337-11e7-9c24-4bcd9a089535.png)

_It use to do it this way before:_
<img width="499" alt="screen shot 2017-03-07 at 12 30 14 pm" src="https://cloud.githubusercontent.com/assets/3847077/23678271/51063306-0338-11e7-8fcd-67e5cf282510.png">

**Comment by davejtoews** [comment]
@elvismdev I just ran into this myself. It's a breaking change in 3.0. Check the README, you now need to manually add paths with the `acfwpcli_fieldgroup_paths` hook.

**Comment by sebastiaandegeus** [comment]
Yes, you need at least 1 path registered. See: https://github.com/hoppinger/advanced-custom-fields-wpcli#acfwpcli_fieldgroup_paths

**Comment by tukera** [comment]
@davejtoews I have the same issues with the export path, can you please tell me where I need to add the `acfwpcli_fieldgroup_paths` hook?

**Comment by davejtoews** [comment]
@tukera, see the link above from @sebastiaandegeus. This should go in your functions.php if you're working in a theme, or in your plugin file if working on a plugin.

**Comment by sampohaavisto** [comment]
This is very confusing. I was reading the code before visiting the issue pages and seeing that it was just a simple hook thing to put the paths visible.

The hook/filter setup should be in the `readme` before the export section and the export section should have a notification about the filter requirement. I'll make a PR for this.

BTW you can also use `wp acf export --export_path=exports/` to manually define the exports path.
